lib/python/treadmill_aws/bootstrap/zookeeper/aws.linux/treadmill/init/zk.yml

```diff
@@ -0,0 +1,23 @@
+# TODO: enable SASL.
+#    -Djava.security.auth.login.config=${ZKROOT}/conf/jaas.conf \
+#    -Dsun.security.krb5.debug=false \
+
+user: "{{ treadmillid }}"
+command: |
+  ZKROOT=/treadmill/zookeeper
+  for J in $(ls {{ zk_distro }}/*.jar); do
+    CP=${CP:+$CP}:$J;
+  done
+  echo CLASSPATH: ${CP}
+  exec \
+    {{ java }} -cp ${CP} \
```

Comment:
You also want to support JVM flags to control java like max memory, option for multithreaded GC (as opposed to "stop the world" which is evil in combination with Zk).